src/libs/OptionsListUtils.js

```diff
@@ -240,6 +240,8 @@ function createOption(personalDetailList, report, draftComments, {
         // It doesn't make sense to provide a login in the case of a report with multiple participants since
         // there isn't any one single login to refer to for a report.
         login: !hasMultipleParticipants ? personalDetail.login : null,
+        phoneNumber: !hasMultipleParticipants ? personalDetail.phoneNumber : null,
+        payPalMeAddress: !hasMultipleParticipants ? personalDetail.payPalMeAddress : null,
```

Comment:
@nickmurray47 I had to add the keys here in `OptionsListUtils` to make it work for the Global create.

Reply:
^ended up adding these keys below so that the button-menu PR would pass QA